devtools/packaging/scripts/android/config_pb2.py

```diff
@@ -0,0 +1,921 @@
+# -*- coding: utf-8 -*-
```

Comment:
How is this file generated? Do we have to have it in the repo, or it could be generated on demand?

If not, then how it should be updated after the `config.proto` file is changed?

Reply:
Yes it can be generated on demand by using protobuf-compiler (cmd: `protoc --python_out . config.proto`).

Reply:
this needs the protobuf compiler though, which is not the same as having installed the python protobuf package.

Reply:
Yes, the protobuf-compiler needs to be installed on the user's pc, which makes the `protoc` command available. The protobuf-compiler is only needed for generating the new _pb2.py file (if config.proto is changed) and is not required for the packaging process. The packaging process requires this _pb2.py file and the python protobuf package (already available in docker-bb).